src/project_creator/templates/project/{{cookiecutter.name}}/doc/test_report_template.txt

```diff
@@ -1,41 +1,38 @@
-**Statistics**
-
-| Test cases: :need_count:`'{id}' in tags and type=='{case_need}'`
-| Failed test cases: :need_count:`'{id}' in tags and 'failure' == result and type=='{case_need}'`
-| Passed test cases: :need_count:`'{id}' in tags and 'passed' == result and type=='{case_need}'`
-| Skipped test cases: :need_count:`'{id}' in tags and 'skipped' == result and type=='{case_need}'`
-| Test suites: :need_count:`'{id}' in tags and type=='{suite_need}'`
-
-**Failed test cases**:
+Passed test cases

```

Comment:
We need to use the `id in tags` condition to filter only the test results relevant for the component.